app/templates/frameworks/edit_declaration_section.html

```diff
@@ -26,6 +26,20 @@
 
 {% block main_content %}
 
+{% with messages = get_flashed_messages() %}
+  {% if messages %}
+    {% for message in messages %}
+      {% if message == 'supplier_declaration_saved' %}
+      <div class="banner-success-without-action">
+        <p class="banner-message">
+          Updates to your supplier declaration so far have been saved
```

Comment:
I don't think we need to give feedback here. It gives a false sense of completeness and isn't something we do for services.